runtime/build.rs

```diff
@@ -4,7 +4,23 @@ fn main() {
 		.with_current_project()
 		.export_heap_base()
 		.import_memory()
-		.build()
+		.build();
```

Comment:
You don't need this. Or should this be the dev chain? Why not make the rococo the dev chain?

Reply:
I thought this is the Polkadot chain? (the default build without feature flags)

Reply:
Added an explicit `polkadot-runtime` feature flag